docs/migration/migrate-odoo-10-to-11.rst

```diff
@@ -38,9 +38,17 @@ New references
     find . -type f -name '*.py' | xargs sed -i "s/'base_action_rule'/'base_automation'/g"
     find . -type f -name '*.py' | xargs sed -i 's/"base_action_rule"/"base_automation"/g'
 
+    # base.config.settings merged into res.config.settings
+    find . -type f -name '*.xml' | xargs sed -i 's/base.config.settings/res.config.settings/g'
+    find . -type f -name '*.py' | xargs sed -i 's/base.config.settings/res.config.settings/g'
```

Comment:
You need to escape dots on left side, I.e. use base\.config...
Also, you can make single line via -or